### DISCUSSION TITLE
Confused ... What's the point?

### DISCUSSION THREAD

**Discussion by ArchieLinux** [anonaddy/anonaddy#537] [Q&A] [answered]
I appreciate the FOSS model embraced by the creator. But I'm confused...

In the free version, you can't respond to emails you receive at your anonaddy alias email addresses. In short, you cannot actually communicate with the recipients from whom you needed to protect your regular email account...

So please help me understand, what's the point of using addy.io's free account?

After all the effort to set it up and test it, it seems that the free version is entirely useless for any reasonable use case. You can receive emails but you cannot reply? How is that useful?

Am I missing something before deleting and uninstalling?

**Comment by JensSpanier** [comment]
A lot of services only send emails and you can’t send mails to them. For example I never answered or wrote an email to Amazon.

**Reply by ArchieLinux** [reply]
Don't get me wrong, I greatly appreciate the FOSS model being embraced here. And just because something doesn't meet my personal "use case" doesn't mean it's not valuable to others...

But the "premium" levels clearly indicate that being able to SEND an email is an essential function that's been disabled in the free version, so there's that to reflect on. 

In short, I cannot think of a good reason to give out an email address from which I cannot initiate a message. Even those services you refer to can send out emails ... they just won't accept replies. That's precisely the opposite of what the free version provides.

I guess I'll simply uninstall in hopes of finding a better FOSS solution some day. 

Thanks again for taking time to reply! :vulcan_salute:

**Reply by roke-julian-lockhart** [reply]
> I cannot think of a good reason to give out an email address from which I cannot initiate a message.

@ArchieLinux, you can respond with your personal e-mail address. I find that a good use case for this is registering for accounts. You'll likely never need to, nor be able to, respond to a message sent by them, and if you do, a personal e-mail account is sufficient.

**Comment by roke-julian-lockhart** [comment]
To respond to https://github.com/anonaddy/anonaddy/discussions/537#discussion-5698809:

> You can receive emails but you cannot reply? How is that useful?

Because people like me don't use it as a privacy tool, just an e-mail consolidator.

**Reply by ccorneli** [reply]
I would even guess that it is how the majority of people use it.  That's my use case too. not just a consolidator, but as a spam preventer..(by quickly killing aliases that get picked up on spam lists).    99% of my email through it is inbound only email notifications or subscriptions.  I rarely ever need to send.

**Reply by ArchieLinux** [reply]
Thanks for that additional background. Perhaps I'm not thinking through the potential use cases properly...

So what kind of recipients are you using the alias accounts for? Probably not any kind of accounts you actively interact (bilaterally) with, so what other kinds what that typically be?

**Reply by roke-julian-lockhart** [reply]
> https://github.com/anonaddy/anonaddy/discussions/537#discussioncomment-7305046

@ArchieLinux, I use it for everything, but whether I continue using the alias in responses depends upon whether retaining any anonymity gained by the address or demonstrating that I'm the same person is unusually important.

**Reply by ccorneli** [reply]
I have given a unique alias to every website I have registered with...

Amazon, Verizon, Zillow, github, Walmart, Best Buy, many news feeds and subscriptions announcement lists for different interest groups, etc.

So as an example: when there is a data breach or my info is sold and i start getting viagra and Xanax emails sent to the alias I gave only to Target, I know where it leaked from and can immediately cut it off by logging into Target to give them a new one.

**Reply by ArchieLinux** [reply]
@ccorneli  Thanks for the additional background. This certainly makes sense as a spam-filtering "firewall" - with the ability to turn on/off individual sources. I never thought of it that way, so this is helpful. 

Logging in with a source from which future communication is still desired and changing to a new alias sounds like an effective way to weed the garden.

I guess I 've been using the same primary email address for so many years (decades) at this point, converting over to individual aliases might take more effort than is worth to avoid the minimal spam-filtering energy I currently face. Food for thought.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/anonaddy/anonaddy/discussions/537#discussioncomment-7340472

@ArchieLinux, I have approximately 2100 online accounts is my password manager. It seemed daunting at first, but the way I do it is simply change the registered address at whatever account I'm currently logged into if it hasn't been yet.

**Reply by ArchieLinux** [reply]
Yikes ... so in effect you have a couple of thousand unique aliases in action? 
Sounds like a daunting effort to implement, but then again it must be worth it if you've got such a broad profile as a spam target.

**Reply by roke-julian-lockhart** [reply]
> https://github.com/anonaddy/anonaddy/discussions/537#discussioncomment-7343228

@ArchieLinux, I've solely managed to get so far:

<p></p><img width="2426" height="126" alt="image" src="https://github.com/user-attachments/assets/f5a140de-c467-4e4a-9535-50e52a77e22e" /><p></p>

**Reply by ArchieLinux** [reply]
Decent progress ;-) You've certainly piqued my further interest! Is it safe to assume you've decided to go with one of the paid plans as opposed to the basic free version?

**Reply by roke-julian-lockhart** [reply]
Yes. I rarely ever pay for anything online because I usually can't. However, it was surprisingly cheap for the throughput I use (although of course my Gmail and Outlook recipient(s) are free to use).

<p></p><img width="2560" height="1907" alt="Screenshot_20250902_154951" src="https://github.com/user-attachments/assets/545ab9e6-67aa-43d7-ae73-7f55bb071483" /><p></p>

Notice the ratio of forwards to replies via AnonAddy - I really don't use it as a privacy tool much.

**Comment by justauserid** [comment]
Am I missing something? I think you can indeed reply from the free plan alias which was actually surprising (and yes amazing). I mean I do love a free feature or two, but seeing the free service (and FOSS on top!) and quite generous I am just jotting it down here that if it's a bug then I'd love the dev to rather fix that and if I would need replies I'd pay for the Lite. Also, I just started few days back and have only sent 2 test mails I am not sure - but I do not see any trial on at all anywhere.

(Anyway tagging @willbrowningme to check)

**Reply by willbrowningme** [reply, marked as answer]
Free accounts can make 1 reply/send in order to test the functionality out, after that you need to be on a paid plan to reply/send from aliases.

**Reply by ArchieLinux** [reply]
That was my understanding from the above exchange. It's an interesting app, but the free account doesn't fit my use case and I can't justify the paid plan in my situation. Alas.

**Reply by justauserid** [reply]
> That was my understanding from the above exchange. It's an interesting app, but the free account doesn't fit my use case and I can't justify the paid plan in my situation. Alas.

SimpleLogin I beleive offers replies and even sending mails from free a/c as well. Just check it out. That might fit your uscase. I do not really use use disposable email services for sending/replying from so this kinda works for me. 

Also increasing I have been relying on <random word but generated by me, so not really gibberish>@mailbox.org for a lot of this, so that works for me as I am a paid user there and I can _retire_ those aliases whenever I want. There's duck.com as well but that's essentially a blackbox and even their paid service are decades behind what addy/SL offer in their free service. Apple's HME (which I sadly have used a lot and now it's taking time in winding down) is worse only to duck.com I believe so yeah that sucks too. 

Anyway I wish you find a service that works for you. Cheers.
